### ISSUE TITLE
Record ghost paths for `kmod`

### ISSUE THREAD

**Issue by ikeycode** [serpent-os/moss#288]
TLDR when we update a kernel the live VFS "forgets" the old, running kernel. This is partially resolved in Serpent OS by baking `vfat` into the `linux-desktop` kernel ensuring that we can always mount ESP & XBOOTLDR.

However for peripherals and general XHCI-land, we still rely on `_request_module` -> `/sbin/depmod` interplay. To that end we'll make `moss` record the ghosted paths of transaction kernel trees under `/run` which redirect to `/.moss/root/$TX/usr/lib/modules/$VERSION`. These shortlived redirects will keep hotplugging working until reboot, upon which the tmpfs `/run` is lost and all redirects are forgotten / unnecessary.
